layouts/_default/list.html

```diff
@@ -1,9 +1,7 @@
-{{ define "main" }}
-  <h1>Posts</h1>
-  {{ range .Pages }}
-    <article>
-      <h2>{{ .Title }}</h2>
-      {{ .Content }}
-    </article>
+<ul>
```

Comment:
This is a basic style add to display each tag in a list and will be updated in a separate PR.